omigami/plotting.py

```diff
@@ -0,0 +1,60 @@
+from PIL.PngImagePlugin import PngImageFile
+from pandas import DataFrame
+from rdkit import Chem
+from rdkit.Chem import Draw
+
+
+def _raise_errors(spectra_matches: DataFrame, representation: str = 'smiles'):
+    if representation not in ["smiles", "inchi"]:
```

Comment:
should we check for compound name too?? not sure though

Reply:
We can either add the compound name here or make the name optional. What do you think?

Reply:
huumm but then how does the user know which structure is which?